src/js/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -0,0 +1,77 @@
+// TODO: remove this hard-coded response once we can fetch from a vets-api endpoint
+const chapter33EnrollmentResponse = {
```

Comment:
Should we call it `post911GibStatusResponse` for consistency with the rest of the code? Even if the contents of the response will be an `chapter33EducationInfo` object